stampifier.py

```diff
@@ -123,14 +123,11 @@ def _summarize(self):
         self.stampified_pages = summarizer.get_summarized_content()
 
     def _get_min_index_for_stamp_page(self, stamp_page):
-        indices = list()
-        if stamp_page.sentence_index != -1:
-            indices.append(stamp_page.sentence_index)
+        if stamp_page.para_index != -1:
+            return stamp_page.get_weighted_text_index()
 
         if stamp_page.media_index != -1:
-            indices.append(stamp_page.media_index)
-
-        return min(indices)
+            return stamp_page.media_index
```

Comment:
Why are we always returning the media index here?

Reply:
As you mentioned in the doc we should give preference to the text index if present. In the case it is not present media should be present and we return its index accordingly! hope this helps.

Reply:
Got it!